packages/lmi/README.md

```diff
@@ -383,6 +384,27 @@ result = await llm.call_single(
 # the calculator invocation with x=2, y=2, operation="+"
 ```
 
+### Vertex
+
+Vertex requires a bit of extra set-up. First, install the extra dependency for auth:
+
+```sh
+pip install google-api-python-client
```

Comment:
Why do we have to install this if your below code snippet doesn't use it?

Reply:
Mainly, if you're using this to auth up front, can you also document the command used with this Python package